sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -341,9 +343,11 @@ def create_job_resources(options,  # type: PipelineOptions
       pickled_session_file = os.path.join(
           temp_dir, names.PICKLED_MAIN_SESSION_FILE)
       pickler.dump_session(pickled_session_file)
-      resources.append(
-          Stager._create_file_stage_to_artifact(
-              pickled_session_file, names.PICKLED_MAIN_SESSION_FILE))
+      # for pickle_library: cloudpickle, dump_session is no op
+      if os.path.exists(pickled_session_file):
```

Comment:
I like the way you did this.  Is it possible to add a unit test for this behavior?

Reply:
Done